django/db/backends/sqlite3/base.py

```diff
@@ -168,9 +169,27 @@ def get_new_connection(self, conn_params):
         conn.create_function("django_timestamp_diff", 2, _sqlite_timestamp_diff)
         conn.create_function("regexp", 2, _sqlite_regexp)
         conn.create_function("django_format_dtdelta", 3, _sqlite_format_dtdelta)
-        conn.create_function("django_power", 2, _sqlite_power)
         conn.create_function('LPAD', 3, _sqlite_lpad)
         conn.create_function('RPAD', 3, _sqlite_rpad)
+        conn.create_function('ACOS', 1, math.acos)
+        conn.create_function('ASIN', 1, math.asin)
+        conn.create_function('ATAN', 1, math.atan)
+        conn.create_function('ATAN2', 2, math.atan2)
+        conn.create_function('CEILING', 1, math.ceil)
+        conn.create_function('COS', 1, math.cos)
+        conn.create_function('COT', 1, lambda x: 1 / math.tan(x))
```

Comment:
I haven’t verified this myself, but [this SO answer](https://stackoverflow.com/a/3738696/1286628) says it’s better to use cos/sin than 1/tan. Intuitively I understand it near pi/2 but I would expect the opposite to be true close to zero. Not sure if there’s a one best option here. Just wanted to raise this in case anyone had opinions.

Reply:
Maybe one option is to split the inputs based on whether they’re closer to +/-pi/2 or +/- pi (mod 2pi).

Reply:
@wkschwartz: Is there a better source for this recommendation to use `lambda x: math.cos(x) / math.sin(x)`?

If we were to make this change, we should add a comment here to explain why.